### PR TITLE
no more edges, no more entity key ids

### PR DESCRIPTION
**Pull request by makosblade**
This PR:
- Removes long unused edges and entity_key_ids table objects

### CODE REVIEW

File: src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt [hunk 1]
```diff
@@ -234,7 +233,7 @@ internal fun buildEntitiesClause(entityKeyIds: Map<UUID, Optional<Set<UUID>>>, l
                 .filter { ids -> ids.isNotEmpty() }
                 .map { ids -> " $idsColumn IN (" + ids.joinToString(",") { id -> "'$id'" } + ") AND" }
                 .orElse("")
-        " ($idsClause ${ENTITY_KEY_IDS.name}.${ENTITY_SET_ID.name} = '${it.key}' )"
+        " ($idsClause entity_key_ids.${ENTITY_SET_ID.name} = '${it.key}' )"
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt:236]: Hmmm...

File: src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt [hunk 2]
```diff
@@ -218,7 +217,7 @@ internal fun selectEntityKeyIdsWithCurrentVersionSubquerySql(
         }
     }
 
-    return "( SELECT $selectColumns FROM ${ENTITY_KEY_IDS.name} INNER JOIN $FILTERED_ENTITY_KEY_IDS USING($joinColumnsSql) WHERE true $entitiesClause $groupBy ) as $ENTITIES_TABLE_ALIAS"
+    return "( SELECT $selectColumns FROM entity_key_ids INNER JOIN $FILTERED_ENTITY_KEY_IDS USING($joinColumnsSql) WHERE true $entitiesClause $groupBy ) as $ENTITIES_TABLE_ALIAS"
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt:220]: I think entity_key_ids replace should have been switched to IDS.name

Comment (makosblade) [reply src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt:220]: This class is appears to be a leaf node in a chain of deprecated/unused/commented code so I just switched it to the historical name for that variable. I can swap this class to IDS but I'm fairly sure it won't work/is unused

Comment (geekbeast) [reply src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt]: Oh right. Good point, we should see if we can delete this.

Comment (geekbeast) [reply src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt]: https://jira.openlattice.com/browse/LATTICE-2547

**Pull request merged**